src/index.js

```diff
@@ -10,7 +10,8 @@ const languages = [
   {
     extensions: ['.sol'],
     name: 'Solidity',
-    parsers: ['solidity-parse']
+    parsers: ['solidity-parse'],
+    vscodeLanguageIds: ['solidity']
```

Comment:
Don't like making a PR based on a pending PR, but it looks like vs-code will ask for this info anyway.

Reply:
can you elaborate a little bit more on this @Janther

Reply:
at the moment there's a PR in prettier-vscode prettier/prettier-vscode#757 waiting for approval.
This PR looks for a local prettier, then a global prettier and if prettier is not installed in the system, it relies on it's own packaged version of prettier.
if you have prettier and our plugin installed locally in the project, it will load our plugin.
then vs-code will call [getSupportInfo](https://prettier.io/docs/en/api.html#prettiergetsupportinfo-version) and there search in the languages for the value `vscodeLanguageIds`. that's why even if prettier/prettier-vscode#757 is not approved, the moment any plugin tries to use prettier as a formatter for vscode, it must provide `vscodeLanguageIds`.

Reply:
that PR was opened the 17th of March hehehe 😅